src/main/java/org/apache/pirk/responder/wideskies/spark/ComputeResponse.java

```diff
@@ -316,23 +311,15 @@ public void performQuery() throws Exception
 
   /**
    * Method to perform the query given an input RDD of MapWritables
-   * 
+   *
    */
   public void performQuery(JavaRDD<MapWritable> inputRDD) throws PIRException
   {
     logger.info("Performing query: ");
 
-    // If we are using distributed exp tables -- Create the expTable file in hdfs for this query, if it doesn't exist
-    if ((queryInfo.getUseHDFSExpLookupTable() || useHDFSLookupTable) && query.getExpFileBasedLookup().isEmpty())
-    {
-      // <queryHash, <<power>,<element^power mod N^2>>
-      JavaPairRDD<Integer,Iterable<Tuple2<Integer,BigInteger>>> expCalculations = ComputeExpLookupTable.computeExpTable(sc, fs, bVars, query, queryInput,
```

Comment:
Is this removal safe?  Although the return value is ignored, the compute method has the side-effect of creating the exponent look-up table in HDFS, and storing it in the Query.